inginious/frontend/plugins/register_students/pages/api/add_course_students_csv_file_api.py

```diff
@@ -120,6 +132,22 @@ def _is_user_email_equal(self, user_email, data_email):
 
         return False
 
+
+    def _email_user_register_on_course(self, username, email, course):
+        """
+        Format an email to an existing user enrolled to a course,
+        :param email: string of the email of the user
+               course: course to enroll
+        :return: a Tuple with the email and the content of the email to send.
+        """
+        data_policy_link = web.ctx.home + "/data_policy"
+        course_link = web.ctx.home + "/course/" + course.get_id() + "/"
```

Comment:
```suggestion
        course_link = web.ctx.home + "/course/" + course.get_id()
```

Remove the last character from course_link url, as courses will not be found with it.